creator-cards/smaicas.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Samuel Maícas",
+  "countryCode": "es",
+  "slogan": "",
+  "type": "CREATOR",
+  "bio": "",
+  "tags": "C#, .NET, BLAZOR, MAUI, ASP.NET ",
+  "socials": {
+    "youtube": "https://www.youtube.com/@cdecsharp",
+    "linkedin": "https://www.linkedin.com/in/samuel-ma%C3%ADcas/",
+    "twitter": "",
+    "mastodon": "",
+    "sessionize": "",
+    "other": ""
+  },
+  "channels": [
+    {
+      "name": "Twitch",
+      "url": "https://www.twitch.tv/viciostv"
+    }
+  ],
+  "feeds": [
+  ],
+  "projects": [
+    {
+      "name": "Dev N Joy Colab",
+      "url": "https://github.com/smaicas-org"
```

Comment:
Can you please change that to the project directly, we can't work with orgs at this point but need project links :)

Reply:
> Can you please change that to the project directly, we can't work with orgs at this point but need project links :)

I've done. Actually project repo is index/wiki about the project but references to my open repository. In fact is open to other users colaboration too. Thanks for the help @DeeJayTC